src/test/java/com/malinatran/request/RequestListenerTest.java

```diff
@@ -4,17 +4,20 @@
 import com.malinatran.mocks.MockRequestReader;
 
 import java.io.IOException;
+
 import org.junit.Test;
 import static org.junit.Assert.*;
 
 public class RequestListenerTest {
 
     @Test
     public void testGetNextRequest() throws IOException {
```

Comment:
Let's get rid of the `test` prefix as it doesn't add any additional meaning to the test names. Additionally, it may look verbose, but let's add more to the test names. What are we testing about `GetNextRequest`? What can I expect this method to do?